Cargo.toml

```diff
@@ -24,6 +24,7 @@ fs4 = "0.6"
 # to use direct functions on FreeBSD
 rustix = { version = "0", features = [ "fs" ]}
 crossbeam-channel = "0.5"
+parking_lot = { version = "0.12.1", features=["deadlock_detection"] }
```

Comment:
I don't think we should enable `deadlock_detection` by default at the library level.
This is a decision for the application developer I believe.

Reply:
Thanks. I fixed it